ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/LedgerFactory.scala

```diff
@@ -93,15 +93,20 @@ trait ConfigProvider[ExtraConfig] {
   def partyConfig(@unused config: Config[ExtraConfig]): PartyConfiguration =
     PartyConfiguration.default
 
-  def initialLedgerConfig(config: Config[ExtraConfig]): InitialLedgerConfiguration =
+  def initialLedgerConfig(config: Config[ExtraConfig]): InitialLedgerConfiguration = {
```

Comment:
For consistency, I'd suggest removing the braces here as we tend to not use them for single expressions.